tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -14,7 +14,7 @@ use warnings;
 use testapi qw(save_screenshot get_var);
 
 sub run {
-    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    my @scc_addons = grep($_, split(/,/, get_required_var('SCC_ADDONS')));
```

Comment:
you will need on top `get_required_var` imported.

Reply:
Updated